test/driver/error_embedding/test.t/run.t

```diff
@@ -0,0 +1,54 @@
+Keep the error output short in order to avoid different error output between different compiler versions in the subsequent tests
+
+  $ export OCAML_ERROR_STYLE=short
+
+With the `-embed-errors` options, if a PPX raises, the first such exception is caught and packed into an output AST
+
+  $ echo "let _ = [%raise]" > impl.ml
+  $ raiser -embed-errors impl.ml
+  [%%ocaml.error "Raising inside the rewriter"]
+
+The same is true when using the `-as-ppx- mode (note that the error is reported by ocaml itself)
```

Comment:
```suggestion
The same is true when using the `-as-ppx` mode (note that the error is reported by ocaml itself)
```

Reply:
For some reason github doesn't let me press commit suggestion. Anyways, done.